stubs/pyOpenSSL/OpenSSL/SSL.pyi

```diff
@@ -155,7 +195,10 @@ class Context:
     def set_keylog_callback(self, callback: Callable[[Connection, bytes], object]) -> None: ...
     def set_alpn_protos(self, protos: Sequence[bytes]) -> None: ...
     def set_alpn_select_callback(self, callback: Callable[[Connection, list[bytes]], bytes]) -> None: ...
-    def set_ocsp_server_callback(self, callback: Callable[[Connection, _T | None], bytes], data: _T | None = ...) -> None: ...
+    def get_alpn_proto_negotiated(self) -> bytes: ...
+    def set_ocsp_server_callback(
+        self, callback: Callable[[Connection, bytes | None], bytes], data: bytes | None = ...
+    ) -> None: ...
     def set_ocsp_client_callback(
-        self, callback: Callable[[Connection, bytes, _T | None], bool], data: _T | None = ...
+        self, callback: Callable[[Connection, bytes, bytes | None], bool], data: bytes | None = ...
     ) -> None: ...
```

Comment:
Looks like you're accidentally reverting #8371 here

Reply:
I will rebase later :)